lib/chitter_manager.rb

```diff
@@ -0,0 +1,38 @@
+require 'pg'
+
+class ChitterManager
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps ORDER BY peep_date DESC;")
+    result.map do |peep| 
+      ChitterManager.new(id: peep['id'], message: peep['message'], date: peep['peep_date'])
+    end
+  end
+
+  def self.post(message:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else 
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
You're using this same piece of code in more than one place in this file. How might you get rid of this duplication?